test/CadlRanchProjects/type/property/optionality/src/Generated/Docs/Plaindate.xml

```diff
@@ -0,0 +1,325 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I updated cadl-ranch-specs to latest version which add more tests, so all the changes in this project are expected.

Reply:
well is `plaindate` a word? This might be a typo, so is `plaintime`. We need to fix this on the cadl ranch side.
In typespec, it is `plainDate` instead of `plaindate`: https://github.com/microsoft/typespec/blob/d157d2e18d7a8887ee723b6d8a2caea65a90d4eb/packages/compiler/lib/intrinsics.tsp#L101